### DISCUSSION TITLE
[Feature] Android app for Fire stick

### DISCUSSION THREAD

**Discussion by RogerHartmann** [immich-app/immich#12684] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The Android app is not well usable on Fire Stick. E.g. with the remote, I cannot go into the timeline and select images.
With a mouse, I can, but I cannot go to the next or previous image (I have to go back to timeline and then click the next image).
With the keyboard, I also cannot.

Suggestion: make the mouse wheel go to the next (resp. previous) image. Make the keyboard cursor do similar.

PS: the web app is not nice on Fire stick, the silk browser keeps showing the title bar.

thanks! Roger

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by danieldietzler** [comment]
The Fire TV app is not from us.

**Reply by RogerHartmann** [reply]
I have actually installed [app-armeabi-v7a-release.apk](https://github.com/immich-app/immich/releases/download/v1.115.0/app-armeabi-v7a-release.apk) on the Fire stick. Not the Fire TV app.
So this is not the Fire TV app, it is the normal immich app from github.

**Reply by danieldietzler** [reply]
Oh I see. We don't target/try to optimize the android app for Fire TVs at the moment though. In fact, we don't test on there or anything. That it works at all is more of a "nice side effect"

**Reply by RogerHartmann** [reply]
I see. I have no idea how much mouse wheel or keyboard cursor support would mean, maybe, it is not that much, it could be implemented one point in time?

**Reply by bo0tzz** [reply]
The browsers on TVs are usually ancient versions, so it's not really feasible for us to support those.

**Reply by RogerHartmann** [reply]
hello @bo0tzz , I just gave the browser as example. The feature request is regarding the normal android app [app-armeabi-v7a-release.apk](https://github.com/immich-app/immich/releases/download/v1.115.0/app-armeabi-v7a-release.apk), to make it support mouse wheel and/or keyboard cursurs to switch between images. That is all
